components/aggregation/mat/StackedBarChart.js

```diff
@@ -22,15 +39,15 @@ export const StackedBarChart = ({ data, query }) => {
       let indexBy = ''
       indexBy = query['axis_x']
       let reshapedData = Array.isArray(data.data.result) ? data.data.result.map(d => {
-        d['ok_count'] = d.measurement_count - d.confirmed_count - d.anomaly_count
+        d['ok_count'] = d.measurement_count - d.anomaly_count - d.failure_count
```

Comment:
It's possible for a measurement to be anomalous, but also be marked as fail. Until https://github.com/ooni/backend/issues/486 is implemented, I think it's best to just subtract `anomaly_count` from the `measurement_count` to compute the `ok_count`.